Cargo.toml

```diff
@@ -26,27 +26,27 @@ default-logger = ["slog-stdlog", "slog-envlogger", "slog-term"]
 # Make sure to synchronize updates with Harness.
 [dependencies]
 fxhash = "0.2.1"
-fail = { version = "0.3", optional = true }
-getset = "0.0.9"
+fail = { version = "0.4", optional = true }
```

Comment:
Perhaps replace it with `thiserror` is better.

Reply:
quick-error -> thiserror, done!

But, do we need to remove the dependency of fail and the feature of failpoints? @BusyJay